tests/test_abandon_command.rs

```diff
@@ -56,11 +56,11 @@ fn test_rebase_branch_with_merge() {
 
     let stdout = test_env.jj_cmd_success(&repo_path, &["abandon", "d"]);
     insta::assert_snapshot!(stdout, @r###"
-    Abandoned commit b7c62f28 d
+    Abandoned commit vruxwmqv b7c62f28 d
```

Comment:
This might be slightly more readable if we put the change id *after* the commit id, but I'm not sure if it's worth doing it differently from `jj log`.

We could write `Parent/Abandoned revision` instead of `Parent/Abandoned commit`, but it might be nice to emphasize that it's the specific commit, as opposed to a change, that gets abandoned. This makes a difference if the commit is hidden.

Reply:
I'm no longer planning to do this.